registries/registry.go

```diff
@@ -122,12 +121,10 @@ func (r Registry) LoadSpecs() ([]*bundle.Spec, int, error) {
 	}
 
 	if len(filteredNames) != 0 {
-		var buffer bytes.Buffer
-		buffer.WriteString("Bundles filtered by white/blacklist filter:")
+		log.Infof("Bundles filtered by white/blacklist filter:")
 		for _, name := range filteredNames {
-			buffer.WriteString(fmt.Sprintf("-> %s", name))
+			log.Infof(fmt.Sprintf("-> %s", name))
```

Comment:
multiple things can be happening when we run bootstrap in the background. This means that all of these filtered names will be in between other logs. I think that it might be better if we just add new lines to the buffer?

Reply:
what about the passing ones? https://github.com/automationbroker/bundle-lib/blob/master/registries/registry.go#L117-L122
```golang
	if len(validNames) != 0 {
		log.Debugf("Bundles passing white/blacklist filter:")
		for _, name := range validNames {
			log.Debugf("-> %s", name)
		}
}
```

basically it's equivalent to what we did with passing. The new line is acceptable and would also work for my initial concern.